packages/api/test/e2e/nft.e2e.ts

```diff
@@ -49,59 +51,150 @@ afterAll(async () => {
 
 describe('NFTs', () => {
   let collectionId;
-  let schema;
-
-  beforeAll(() => {
-    // setup test collection
-    collectionId = 'example-collection';
-    schema = [
-      ['name', 'text'],
-      ['fingerprint', 'hash'],
-      ['created', 'timestamp']
-    ];
-  });
 
   it('creates a collection', async done => {
-    await api.tx.nft.createCollection(collectionId, schema, "https://example.com/nft/metadata", null).signAndSend(collectionOwner, async ({ status, events }) => {
+    let collectionName = 'example-collection';
+    await api.tx.nft.createCollection(
+      collectionName,
+      {"Https": "example.com/nft/metadata" },
+      null,
+    ).signAndSend(collectionOwner, async ({ status, events }) => {
       if (status.isInBlock) {
         events.forEach(({phase, event: {data, method, section}}) => {
           console.log('\t', phase.toString(), `: ${section}.${method}`, data.toString());
+          if (method == 'CreateCollection') {
+            collectionId = data[0];
+            console.log(`got collection: ${collectionId}`);
+          }
         });
         expect((await api.query.nft.collectionOwner(collectionId)).toString()).toBe(collectionOwner.address);
+        expect((await api.query.nft.collectionName(collectionId)).toString()).toBe(stringToHex(collectionName));
         done();
       }
     });
   });
 
   it('creates a token', async done => {
     const attributes = [
-      {'text': 'hello world'},
-      {'hash': blake2AsHex(stringToU8a('hello world'))},
-      {'timestamp': 12345}
+      {'Text': '💎🙌'},
     ];
 
-    await api.tx.nft.createToken(collectionId, tokenOwner.address, attributes, null).signAndSend(collectionOwner, async ({ status, events }) => {
+    let tokenId;
+    await api.tx.nft.mintUnique(collectionId, tokenOwner.address, attributes, null).signAndSend(collectionOwner, async ({ status, events }) => {
       if (status.isInBlock) {
-        events.forEach(({phase, event: {data, method, section}}) => {
-          console.log('\t', phase.toString(), `: ${section}.${method}`, data.toString());
+        events.forEach(({ event: {data, method }}) => {
+          if (method == 'CreateToken') {
+            tokenId = data[1];
+            console.log(`got token: ${tokenId}`);
+          }
         });
-        // first token has id 0
-        let tokenAttributes = (await api.query.nft.tokenAttributes(collectionId, 0));
-        expect(tokenAttributes.toJSON()).toEqual(attributes);
-        expect((await api.query.nft.tokenOwner(collectionId, 0)).toString()).toBe(tokenOwner.address);
+        let seriesId = tokenId[1];
+        let seriesAttributes = (await api.query.nft.seriesAttributes(collectionId, seriesId));
```

Comment:
this failed on my local with 
Error: nft.seriesAttributes(CollectionId, SeriesId) is a doublemap, requiring 2 arguments, 1 found